bots/bots/aggregation-bot.js

```diff
@@ -13,29 +17,53 @@ class AggregationBot {
    */
   async main() {
     const measurables = await this.api.measurables();
```

Comment:
It's important to only get measurables that have competitive entrees that are newer than aggregates by this specific bot. Can that be added here, or with new stories?

Reply:
I think would be better in another issue

Reply:
ok, plz review this new issue.
https://github.com/OAGr/foretold/issues/187